arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "version": "2017-06-01",
```

Comment:
Current version for private preview release is 2017-01-01. For GA we will be using the specified version 2017-06-01. Hence, we haven't added the suffix "-preview".